modules/controlplane/main.tf

```
@@ -728,6 +734,21 @@ resource "aws_ecs_service" "control_plane_service" {
 
   desired_count = var.desired_task_count
 
+
+
+  service_connect_configuration {
+    enabled   = true
+    namespace = var.service_discovery_namespace_arn
+    service {
+      discovery_name = "control_plane-grpc"
+      port_name      = "control_plane"
+      client_alias {
+        port     = 8080
+        dns_name = "control_plane.grpc"
```

I'm thinking it might be nice, though maybe unneccesary to prefix these with the service discovery namespace

```suggestion
        dns_name = "{var.service_connect_namespace}.control_plane.grpc"
```

---

I feel like this may be unnecessary. If needed you can see which services are in a particular namespace in ecs or cloudmap